frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/SafeguardForm/SafeguardForm.tsx

```diff
@@ -208,11 +211,27 @@ export const SafeguardForm = ({
 
     const showButtons = mode === 'create' || mode === 'edit';
 
+    const handleDelete = () => {
+        if (onDelete) {
+            onDelete();
+        }
+    };
+
```

Comment:
For progression deletion we have confirmation box. Lets add it here also.

Reply:
good idea. will add in the next pr